doc/flexbench/docs/GSoC/2024/2024.md

```diff
@@ -0,0 +1,112 @@
+# 2024
+
+**Mentors**: Giannis Skitsas, Marios Karagiannopoulos
+
+**Assignee**: Yujun Liu
+
+## Abstract
+
+Flexbench, initially forked from the http-traffic-simulator npm package, has undergone significant enhancements. The initial development during GSoC 2022 transformed the package into a stand-alone desktop application and web server with API exposure, integrating OpenIDC for authentication and authorization and deploying it as a containerized application on Kubernetes. My work during GSoC 2024 further advances Flexbench by implementing a robust data anonymization feature, integrating OpenAPI support for streamlined API interactions, and addressing various UI/UX improvements for the desktop app to ensure a more intuitive user experience.
```

Comment:
@AJun01  I do not remember integrating OpenIDC, did you find relevant implementation?